cfme/containers/node.py

```diff
@@ -120,116 +149,79 @@ def all(self):
 # Still registering Node to keep on consistency on container objects navigations
 @navigator.register(Node, 'All')
 @navigator.register(NodeCollection, 'All')
-class All(CFMENavigateStep):
+class NodeAll(CFMENavigateStep):
     VIEW = NodeAllView
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
     def step(self, *args, **kwargs):
+        """Navigate to the Nodes page"""
         self.prerequisite_view.navigation.select('Compute', 'Containers', 'Container Nodes')
 
-    def resetter(self):
```

Comment:
I think the resetter is important, we assume we get List View by default for each details page.
I suggest:
```python
def resetter(self):
    from cfme.web_ui import paginator
    self.view.toolbar.view_selector.select('Grid View')
    paginator.check_all()
    paginator.uncheck_all()
```

Reply:
If ressetter is necessary, I would argue to use view.entities.paginator.check_all() which is imported from BaseEntitiesView. BaseEnitiesView is conditionally set the entities based on the view selection in the UI so that you can select an entity regardless of the view. I can keep the check_all() and uncheck_all() in the resetter though

Reply:
I will add this back in